crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -0,0 +1,377 @@
+use masking::PeekInterface;
+use serde::{Deserialize, Serialize};
+use time::PrimitiveDateTime;
+use url::Url;
+use uuid::Uuid;
+
+use crate::{
+    core::errors,
+    pii::{self, Secret},
+    services,
+    types::{self, api, storage::enums},
+};
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexIntentRequest {
+    request_id: String,
```

Comment:
Add comments at-least for id's, what is that id and why it is used.

Reply:
Do the same for all other request and response structs also

Reply:
Done.